packages/terra-dropdown-button/src/_DropdownList.jsx

```diff
@@ -185,7 +190,7 @@ class DropdownList extends React.Component {
         onFocus={this.handleFocus}
         onKeyDown={this.handleKeyDown}
         onKeyUp={this.handleKeyUp}
-        role="menu"
+        role="listbox"
```

Comment:
```suggestion
        role="menubar"
```
I think role menubar would be more appropriate for dropdown button : https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menubar_role